include/openenclave/internal/trace.h

```diff
@@ -19,6 +19,9 @@ extern oe_log_level_t _log_level;
 
 #if !defined(OE_BUILD_ENCLAVE)
 oe_result_t oe_log_enclave_init(oe_enclave_t* enclave);
+oe_result_t oe_log_set_enclave_log_level(
```

Comment:
Why do we need to declare this twice?

Reply:
Since this is the public facing header, I had added the API declaration here too. Removed it since it could cause confusion in codebase.